.travis.yml

```diff
@@ -18,6 +18,12 @@ matrix:
       env: COMPOSER_ARGS=""
     - php: "8.0"
       env: COMPOSER_ARGS=""
+    - php: "8.1"
```

Comment:
I'm not sure if Travis CI is working for this repository. I would suggest migrating to GitHub Actions, but we can do that with a separate PR.

Reply:
Removed Travis CI changes